scripts/sam.py

```diff
@@ -183,9 +222,16 @@ def create_mask_batch_output(
             output_blend.save(os.path.join(dino_batch_dest_dir, f"{filename}_{idx}_blend{ext}"))
 
 
+def parse_points(points):
+    if isinstance(points, str):
+        points = eval(points)  # Convert string representation of list back to list
```

Comment:
Is there any specific reason for the `eval` ? It's not great for security reasons. In theory this should be json compliant and `json.loads` should do the trick.

```suggestion
        points = json.loads(points)  # Convert string representation of list back to list
```